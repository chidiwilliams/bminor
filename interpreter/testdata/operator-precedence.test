a: integer = 1 + 4 - -89 / 3 / 4 * 4 - 2;
b: integer = 37 % 89 + (47 - 36) * 3 - 80 ^ 3;
c: boolean = (a <= 1) || (b <= 3) && (a < b);
d: boolean = c || a >= b;
e: boolean = b != b - 1;
f: boolean = a++ < b--;

print a, b, c, d, e, f, "\n";
