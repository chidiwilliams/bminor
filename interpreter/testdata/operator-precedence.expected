32-511931falsetruetruefalse
